include/bluetooth/iso.h

```diff
@@ -138,6 +138,15 @@ struct bt_iso_chan_path {
 	uint8_t				cc[0];
 };
 
+/** @brief ISO Meta Data structure for received ISO packets. */
+struct bt_iso_recv_info {
+	/** ISO timestamp - valid only if the Bluetooth controller includes it */
+	uint32_t ts;
```

Comment:
What is the value if the controller doens't include it? 0? 0xFFFFFFFF?

Reply:
0 - I have update the text. (I have not changed this behavior in this PR)